surveys/sphinx-survey-2025.yml

```
@@ -205,6 +205,8 @@ questions:
       - "Read The Docs"
       - "Pydata Sphinx Theme"
       - "Material design"
+      - "Furo"
+      - "Python Docs Sphinx Theme"
```

* https://github.com/lepture/shibuya
* https://github.com/executablebooks/sphinx-book-theme
* https://github.com/jbms/sphinx-immaterial
* https://github.com/kai687/sphinxawesome-theme
* https://github.com/ryan-roemer/sphinx-bootstrap-theme
```suggestion
      - "Python Docs Sphinx Theme"
      - "Sphinx Book Theme"
      - "Shibuya"
      - "Sphinx Immaterial"
      - "Sphinx Awesome Theme"
      - "Sphinx Bootstrap Theme"
```